geomesa-tools/conf/sfts/adsbx/reference.conf

```diff
@@ -0,0 +1,248 @@
+caches = {
+  spdtyp_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Ground Speed" },
+      "1" = { code = "Ground Speed Reversing" },
+      "2" = { code = "Indicated Air Speed" },
+      "3" = { code = "True Air Speed" }
+    }
+  }
+  wtc_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Light" },
+      "2" = { code = "Medium" },
+      "3" = { code = "Heavy" }
+    }
+  }
+  species_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Land Plane" },
+      "2" = { code = "Sea Plane" },
+      "3" = { code = "Amphibian" },
+      "4" = { code = "Helicopter" },
+      "5" = { code = "Gyrocopter" },
+      "6" = { code = "Tiltwing" },
+      "7" = { code = "Ground Vehicle" },
+      "8" = { code = "Tower" }
+    }
+  }
+  engtype_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Piston" },
+      "2" = { code = "Turboprop" },
+      "3" = { code = "Jet" },
+      "4" = { code = "Electric" }
+    }
+  }
+  engmount_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Unknown" },
+      "1" = { code = "Aft Mounted" },
+      "2" = { code = "Wing Buried" },
+      "3" = { code = "Fuselage Buried" },
+      "4" = { code = "Nose Mounted" },
+      "5" = { code = "Wing Mounted" }
+    }
+  }
+  trt_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Unknown" },
+      "1" = { code = "Mode-S" },
+      "2" = { code = "ADS-B (unknown version)" },
+      "3" = { code = "ADS-B 0 – DO-260" },
+      "4" = { code = "ADS-B 1 – DO-260 (A)" },
+      "5" = { code = "ADS-B 2 – DO-260 (B)" }
+    }
+  }
+  tt_codes = {
+    type = "simple"
+    data = {
+      "a" = { code = "Altitude" },
+      "s" = { code = "Speed" }
+    }
+  }
+  vsit_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Barometric" },
+      "1" = { code = "Geometric" }
+    }
+  }
+  adsbx_static = {
+    type = "redis"
+    redis-url = "localhost"
+  }
+}
+
+geomesa {
+  sfts {
+    adsbx = {
+      user-data {
+        "geomesa.indices.enabled"           = "attr,z3"
+        "geomesa.table.compression.enabled" = "true"
+        "geomesa.table.compression.type"    = "snappy"
+        "parquet.summary.metadata.level"    = "ALL"
+        "parquet.compression"               = "SNAPPY"
+        "geomesa.table.sharing"             = "false"
+        "geomesa.attr.splits"               = "4"
+        "geomesa.index.tiering"             = "date"
+        "geomesa.z.splits"                  = "4"
```

Comment:
we probably want to get rid of most of these - they seem deployment specific and not really related to the data type per say.

Reply:
Are there any that should be retained?